### ISSUE TITLE
About got10k result.

### ISSUE THREAD

**Issue by shi-lt** [botaoye/OSTrack#15]
Hello, I use a single GPU to train the got10k model, the configuration file is **vitb_256_mae_ce_32x4_got10k_ep100**, but the result of the last ten epochs is only **69.x**, is this because of the single GPUtraining?

**Comment by botaoye** [comment]
As the batch size changes, you need to change the initial learning rate accordingly. For example, suppose your batch size is 32, which is 1/4 of the original batch size, you should also change the initial learning rate to 0.0001.

**Comment by shi-lt** [comment]
Thank you very much!
